lib/src/components/servo/service.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/servo/servo.dart';
+import 'package:viam_sdk/src/gen/common/v1/common.pb.dart';
+import 'package:viam_sdk/src/gen/component/servo/v1/servo.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+
+class ServoService extends ServoServiceBase {
+  final ResourceManager _manager;
+
+  ServoService(this._manager);
+
+  Servo _servoFromManager(String name) {
+    try {
+      return _manager.getResource(Servo.getResourceName(name));
+    } catch (e) {
+      // TODO make rescource not found GRPC error
+      print('Exception details:\n $e');
+      rethrow;
+    }
+  }
+
+  @override
+  Future<MoveResponse> move(ServiceCall call, MoveRequest request) async {
+    Servo servo = _servoFromManager(request.name);
+    await servo.move(request.angleDeg, extra: request.extra.toMap());
+    return MoveResponse();
+  }
+
+  @override
+  Future<DoCommandResponse> doCommand(ServiceCall call, DoCommandRequest request) async {
+    // TODO: implement doCommand
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<GetPositionResponse> getPosition(ServiceCall call, GetPositionRequest request) async {
```

Comment:
`get...` !! 

Thinking out loud here...

We don't have the same layer of abstraction here as we do on the client side, so this name has to stay unless we add a layer 🤔 is that worth it? We'll run into this more later with other resource services as well.

Reply:
> so this name has to stay unless we add a layer

Confused what you mean here. Can you not use the method name `position` because of a name spacing issue?